src/Stratis.Bitcoin.Features.Wallet/WalletRPCController.cs

```diff
@@ -221,6 +225,46 @@ public GetTransactionModel GetTransaction(string txid)
             return model;
         }
 
+        [ActionName("listunspent")]
+        [ActionDescription("Returns an array of unspent transaction outputs belonging to this wallet.")]
+        public UnspentCoinModel[] ListUnspent(int minConfirmations = 1, int maxConfirmations = 9999999, string addressesJson = null)
```

Comment:
Is this method not the same as what we have in `WalletManager.GetSpendableTransactionsInAccount`?
Look also at `Walletcontroller.GetSpendableTransactions`.
I think the work might be a bit duplicated here and we could use the aforementioned for our purpose.

Reply:
Yes the logic is almost identical to the API spendable-transactions but formatting specific to the RPC specification.

Reply:
Ah yes indeed.
In this case, can we add the number of confirmations to `UnspentOutputReference` and update `walletManager.GetSpendableTransactionsInAccount` all the way down to `HdAccount.GetSpendableTransactions` ?
Then, if we're missing something in the `UnspentOutputReference` we can add to it.